### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Tagadda**
## Description

- Upgrade to 4.1.0 https://github.com/YunoHost-Apps/mobilizon_ynh/pull/190
- Fix emails thanks to @magostinelli https://github.com/YunoHost-Apps/mobilizon_ynh/pull/191

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Tagadda** [comment]
!testme

**Comment by rodinux** [comment]
Hello, I have try this update, since this update when I connect to the account, I am not redirected to a session connected, I have to refresh the page with `Ctl + R` to acces to a connected session... Perhaps also because of my particular settings to accept no Yunohost users ?

```
config :mobilizon, Mobilizon.Service.Auth.LDAPAuthenticator, Mobilizon.Service.Auth.Authenticator
```
It is like if I have done this

```
config :mobilizon, :ldap,
  enabled: false,
```

**Comment by rodinux** [comment]
> Hello, I have try this update, since this update when I connect to the account, I am not redirected to a session connected, I have to refresh the page with `Ctl + R` to acces to a connected session... Perhaps also because of my particular settings to accept no Yunohost users ?

It's a bug from Mobilizon in fact, not Yunohost... I don't remember where I could something about this...

**Comment by rodinux** [comment]
I thought this PR was pushed on master...

**Comment by Salamandar** [comment]
Rebased to fix merge conflict on readmes.

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by rodinux** [comment]
!testme

**Comment by rodinux** [comment]
By the power of systemd, I invoke The Great App CI to test this Pull Request!

File: scripts/upgrade [hunk 1]
```diff
@@ -23,6 +23,15 @@ ynh_systemd_action --service_name=$app --action="stop" --log_path=systemd
 #=================================================
 ynh_script_progression --message="Ensuring downward compatibility..."
 
+if [ "$debian" == "bookworm" ]; then
+    if ynh_app_upgrading_from_version_before_or_equal_to 4.0.2~ynh1; then
+    ynh_die --message="Please consider first upgrade mobilizon to version 4.1.0~ynh1"
```
Comment (ericgaspar) [inline scripts/upgrade:28]:
```suggestion
    ynh_die --message="Please consider first upgrade Mobilizon to version 4.1.0~ynh1"
```

File: scripts/upgrade [hunk 2]
```diff
@@ -23,6 +23,15 @@ ynh_systemd_action --service_name=$app --action="stop" --log_path=systemd
...
+    fi
+    if [ "$YNH_ARCH" == "arm64" ]; then
+    ynh_die --message="No more package arm64"
```
Comment (ericgaspar) [inline scripts/upgrade:31]:
```suggestion
    ynh_die --message="No more arm64 package"
```

**Review by rodinux — COMMENTED** [review]
LGTM

File: scripts/upgrade (hunk 2 above)
Comment (rodinux) [reply scripts/upgrade]: @ericgaspar , Is ok if I merge this...

**Pull request merged**